src/VolumeDrawable.ts

```diff
@@ -297,6 +306,26 @@ export default class VolumeDrawable {
     this.volumeRendering.setAxisClip(axis, minval, maxval, isOrthoAxis || false);
   }
 
+  // TODO: Change mode to an enum
+  /**
+   * Sets the camera mode of the VolumeDrawable.
+   * @param mode Mode can be "3D", or "XY" or "Z", or "YZ" or "X", or "XZ" or "Y".
+   */
+  setViewMode(mode: string): void {
+    this.viewMode = mode;
+    if (mode === "XY" || mode === "Z") {
+      // If currently in 3D raymarch mode, hotswap the 2D slice
+      if (this.rayMarchedAtlasVolume && !this.atlas2DSlice) {
```

Comment:
Currently it will just show an orthographic z-view in pathtrace mode. Switching to raymarched rendering from pathtraced rendering (or vice-versa) will still show the slice correctly.

Should we show the 2D slice in pathtrace mode as well?
![image](https://github.com/allen-cell-animated/volume-viewer/assets/30200665/8955bdb2-a444-438e-ad6b-5acb0c88adbf)

Reply:
Thanks for checking. That behavior seems fine to me.

Reply:
It might make sense to create a ticket for always using the sliceImpl for XY mode no matter what.  But no need to change it in this PR just yet.